static/js/message_list.js

```diff
@@ -68,14 +68,14 @@ export class MessageList {
             render_info = this.append_to_view(bottom_messages, opts);
         }
 
-        if (this.narrowed && !this.empty()) {
+        if (!this.empty()) {
             // If adding some new messages to the message tables caused
             // our current narrow to no longer be empty, hide the empty
             // feed placeholder text.
             narrow_banner.hide_empty_narrow_message();
         }
 
-        if (this.narrowed && !this.empty() && this.selected_id() === -1) {
+        if (!this.empty() && this.selected_id() === -1) {
```

Comment:
I don't think this line of change is needed since this doesn't affect the narrow_banner